src/OSPSuite.Core/Domain/Builder/PathAndValueEntityBuildingBlock.cs

```diff
@@ -50,7 +50,7 @@ IEnumerator IEnumerable.GetEnumerator()
       public override void AcceptVisitor(IVisitor visitor)
       {
          base.AcceptVisitor(visitor);
-         _allValues.Each(initialCondition => initialCondition.AcceptVisitor(visitor));
+         _allValues.Each(pathAndValueEntity => pathAndValueEntity.AcceptVisitor(visitor));
```

Comment:
Personal thoughts for discussion: not sure big long name in lambdas make it easier to read. Above you used ic, ok. x, for the win in this case (or v for value)